Libplanet/Net/Swarm.cs

```diff
@@ -407,6 +407,17 @@ public void Dispose()
                 TimeSpan.FromMilliseconds(findPeerTimeout),
                 depth,
                 cancellationToken);
+
+            if (!((NetMQTransport)Transport).Running)
+            {
+                // Mark all peers as stale if bootstrap is called before transport is running
+                // FIXME: Should be mark as stale only the peers which are added during
+                // bootstrap step.
+                foreach (var peer in RoutingTable.Peers)
```

Comment:
How about specifying the type of `peer`?

Reply:
Do you mean this?
```suggestion
                foreach (BoundPeer peer in RoutingTable.Peers)
```

Reply:
Yes